packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -78,9 +79,13 @@ export class GraphQLGenerator {
 
   public readModelResolverBuilder(
     readModelClass: AnyClass
-  ): GraphQLFieldResolver<any, GraphQLResolverContext, { filter: Record<string, ReadModelPropertyFilter> }> {
+  ): GraphQLFieldResolver<any, GraphQLResolverContext, ReadModelRequestArgs> {
     return (parent, args, context, info) => {
-      const readModelEnvelope = toReadModelRequestEnvelope(readModelClass.name, args, context)
+      let isPaginated = false
+      if (info.fieldName && info.fieldName.startsWith('List') && info.returnType.toString().endsWith('Connection')) {
+        isPaginated = true
+      }
+      const readModelEnvelope = toReadModelRequestEnvelope(readModelClass.name, args, context, isPaginated)
```

Comment:
```suggestion
      const isPaginated = info.fieldName && info.fieldName.startsWith('List') && info.returnType.toString().endsWith('Connection'))
      const readModelEnvelope = toReadModelRequestEnvelope(readModelClass.name, args, context, isPaginated)
```

Reply:
I changed to a cleaner version, but I'd prefer to leave the conditional, so it get easier to recognise for removing it on the future